packages/datadog-plugin-aerospike/test/index.spec.js

```diff
@@ -0,0 +1,304 @@
+                  expect(span).to.have.property('service', expectedSchema.command.serviceName)
+                  expect(span).to.have.property('resource', `Query`)
+                  expect(span).to.have.property('type', 'aerospike')
+                  expect(span.meta).to.have.property('span.kind', 'client')
+                  expect(span.meta).to.have.property('aerospike.namespace', ns)
+                  expect(span.meta).to.have.property('aerospike.setname', set)
+                  expect(span.meta).to.have.property('component', 'aerospike')
+                })
+                .then(done)
+                .catch(done)
+
+              client.connect(() => {
+                const index = {
+                  ns: ns,
+                  set: 'demo',
+                  bin: 'tags',
+                  index: 'tags_idx',
+                  datatype: aerospike.indexDataType.STRING
+                }
+                client.createIndex(index, (error, job) => {
+                  job.wait((waitError) => {
+                    const query = client.query(ns, 'demo')
+                    const queryPolicy = {
+                      totalTimeout: 10000
+                    }
+                    query.select('id', 'tags')
+                    query.where(aerospike.filter.contains('tags', 'green', aerospike.indexType.LIST))
+                    const stream = query.foreach(queryPolicy)
+                    stream.on('end', () => {})
+                  })
+                })
+              })
+            })
+          }
+          it('should run the callback in the parent context', done => {
+            const obj = {}
+            client.connect(() => {
+              tracer.scope().activate(obj, () => {
+                client.put(key, { i: 123 }, () => {
+                  expect(tracer.scope().active()).to.equal(obj)
+                  done()
+                })
+              })
+            })
+          })
+
+          it('should handle errors', done => {
+            let error
+
+            agent
+              .use(traces => {
+                expect(traces[0][0].meta).to.have.property(ERROR_TYPE, error.name)
+                expect(traces[0][0].meta).to.have.property(ERROR_MESSAGE, error.message)
+                expect(traces[0][0].meta).to.have.property(ERROR_STACK, error.stack)
+                expect(traces[0][0].meta).to.have.property('component', 'aerospike')
+              })
+              .then(done)
+              .catch(done)
+
+            client.connect(() => {
+              client.put(key, { i: 'not_a_number' }, () => {
+                const ops = [
+                  aerospike.operations.incr('i', 1),
+                  aerospike.operations.read('i')
+                ]
+
+                client.operate(key, ops, (err) => {
+                  error = err
+                })
+              })
+            })
+          })
+        })
+      })
```

Comment:
Can we have a `withNamingSchema` check without configuration to verify that we're correctly falling back onto whatever the tracer service is when we don't have more specific configuration?